test/test_helm_python_django_app.py

```diff
@@ -7,13 +7,35 @@
 
 test_dir = Path(os.path.abspath(os.path.dirname(__file__)))
 
+VERSION = os.getenv("VERSION")
+IMAGE_NAME = os.getenv("IMAGE_NAME")
+OS = os.getenv("TARGET")
+
+TAGS = {
+    "rhel8": "-ubi8",
+    "rhel9": "-ubi9",
+    "rhel10": "-ubi10",
+}
+TAG = TAGS.get(OS, None)
```

Comment:
`None` is the default value for `.get` dict method.

```suggestion
TAG = TAGS.get(OS)
```